cmake/version.cmake

```diff
@@ -8,53 +8,57 @@ unset(CCF_VERSION_SUFFIX)
 set(CCF_PROJECT "ccf_${COMPILE_TARGET}")
 
 # If possible, deduce project version from git environment
-if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
```

Comment:
>if [](http://example.com/codeflow?start=0&length=2)

Ignore this file. for setting the constant version for private testing image creation.